helmfile.d/values/grafana/grafana-user.yaml.gotmpl

```diff
@@ -60,8 +60,8 @@ datasources:
       If only one workload cluster, don't bother naming the datasource and set to default.
       Otherwise set the name of the clusters so users know what they are looking at.
       */}}
-    {{- if eq (len .Values.global.clustersMonitoring) 1 }}
-    - name: "Workload Cluster"
+    {{- with first .Values.global.clustersMonitoring }}
+    - name: "Workload Cluster {{ . }}"
```

Comment:
For me I think this will look a bit strange for others using the cluster since this will be something like `Workload Cluster environment-name-wc` for basically all that have single WC.
In that case I'd probably be happier if we just used `{{ . }}` directly.

Also I think that we don't need the separate loops with this setup, they should be identical

Reply:
I'd argue that the user experience would be better with the "Workload Cluster", since we also expose the "Service Cluster" data source.
Additionally the split is mainly because we need to set one as default, else it will be wild west which Grafana will show first.

Reply:
Yea, I also think that that would be the best. If there's only one that is. So basically the same behaviour as we had it?

Right, I missed that one, so you're correct that we need them split

Reply:
I realized that this needs a migration script btw, grafana does not delete old data sources even if you remove them from the list

Reply:
> So should I try to rework it so that if the length is 1 that its just "Workload Cluster" or is it fine to have "Workload Cluster env-name" even with just 1 wc?

I think that would be nice if you could do (Keep "Workload Cluster" if it's only one)

Reply:
alright done, then those clusters won't need the migration step either